dispatches/models/fossil_case/properties/test_thermaloil_properties.py

```diff
@@ -0,0 +1,223 @@
+        assert isinstance(model.props[1].enthalpy_eq, Constraint)
+        assert isinstance(model.props[1].cp_mass, Expression)
+        assert isinstance(model.props[1].density, Expression)
+        assert isinstance(model.props[1].visc_kin, Expression)
+        assert isinstance(model.props[1].therm_cond, Expression)
+
+    @pytest.mark.unit
+    def test_get_material_flow_terms(self, model):
+        for p in model.params.phase_list:
+            for j in model.params.component_list:
+                assert str(
+                    model.props[1].get_material_flow_terms(p, j)) == str(
+                    model.props[1].flow_mass)
+
+    @pytest.mark.unit
+    def test_get_enthalpy_flow_terms(self, model):
+        for p in model.params.phase_list:
+            assert str(model.props[1].get_enthalpy_flow_terms(p)) == str(
+                    model.props[1].enthalpy_flow_terms[p])
+
+    @pytest.mark.unit
+    def test_default_material_balance_type(self, model):
+        assert model.props[1].default_material_balance_type() == \
+            MaterialBalanceType.componentTotal
+
+    @pytest.mark.unit
+    def test_default_energy_balance_type(self, model):
+        assert model.props[1].default_energy_balance_type() == \
+            EnergyBalanceType.enthalpyTotal
+
+    @pytest.mark.unit
+    def test_define_state_vars(self, model):
+        sv = model.props[1].define_state_vars()
+
+        assert len(sv) == 3
+        for i in sv:
+            assert i in ["flow_mass",
+                         "temperature",
+                         "pressure"]
+
+    @pytest.mark.unit
+    def test_define_port_members(self, model):
+        sv = model.props[1].define_state_vars()
+
+        assert len(sv) == 3
+        for i in sv:
+            assert i in ["flow_mass",
+                         "temperature",
+                         "pressure"]
+
+    @pytest.mark.unit
+    def test_initialize(self, model):
+        assert not model.props[1].flow_mass.fixed
+        assert not model.props[1].temperature.fixed
+        assert not model.props[1].pressure.fixed
+
+        model.props.initialize(hold_state=False, outlvl=idaeslog.INFO)
+
+        assert not model.props[1].flow_mass.fixed
+        assert not model.props[1].temperature.fixed
+        assert not model.props[1].pressure.fixed
+
+    @pytest.mark.unit
+    def test_initialize_hold(self, model):
+        assert not model.props[1].flow_mass.fixed
+        assert not model.props[1].temperature.fixed
+        assert not model.props[1].pressure.fixed
+
+        flags = model.props.initialize(hold_state=True)
+
+        assert model.props[1].flow_mass.fixed
+        assert model.props[1].temperature.fixed
+        assert model.props[1].pressure.fixed
+
+        model.props.release_state(flags, outlvl=idaeslog.INFO)
+
+        assert not model.props[1].flow_mass.fixed
+        assert not model.props[1].temperature.fixed
+        assert not model.props[1].pressure.fixed
+
+    @pytest.mark.component
```

Comment:
I don't know what the difference between `@pytest.mark.unit` and `@pytest.mark.component` is. Having said that, should it be `@pytest.mark.unit` instead (consistent with the previous methods)?

Reply:
updated for consistency